src/backend/dx11/src/data.rs

```diff
@@ -205,19 +206,21 @@ pub fn map_bind(bind: Bind) -> D3D11_BIND_FLAG {
 }
 
 
-pub fn map_access(access: MapAccess) -> D3D11_CPU_ACCESS_FLAG {
+pub fn map_access(access: mapping::Access) -> D3D11_CPU_ACCESS_FLAG {
     match access {
-        MapAccess::Readable => D3D11_CPU_ACCESS_READ,
-        MapAccess::Writable => D3D11_CPU_ACCESS_WRITE,
-        MapAccess::RW => D3D11_CPU_ACCESS_READ | D3D11_CPU_ACCESS_WRITE,
+        mapping::READABLE => D3D11_CPU_ACCESS_READ,
```

Comment:
I think `READABLE` and `WRITABLE` are against the code style conventions of Rust. Perhaps, we could leave them as `Readable`/`Writable`?